### DISCUSSION TITLE
Is there any way to find or get a node that's centroid of a network graph?

### DISCUSSION THREAD

**Discussion by yothgewalt** [reaviz/reagraph#260] [Q&A]
Is there any way to find or get a node that's centroid of a network graph?

**Comment by ghsteff** [comment]
This reminds me of the [graphology metrics](https://graphology.github.io/standard-library/metrics.html#degree-centrality) we use to compute sizing - [here's the sizing story](http://localhost:9009/?path=/docs/docs-advanced-sizing--docs)

Maybe we could add a `getNodes()` callback or something that you can use to get a list of nodes and their "rankings" without affecting sizing? We'd prob also need to add somethin like a `ranking` prop, similar to `sizingType` 

e.g.
```
const nodes = graphRef.current.getNodes();

<GraphCanvas ref={graphRef} ranking="centrality"/>
```
